### PR TITLE
Riley nitpicks

### PR DESCRIPTION
**Pull request by grandpariley**

### CODE REVIEW

File: components/image_centered.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ...rest
+  }: ICenteredImage) => {
+    return (
+        <div>
+        <div className="w-screen h-1/6"></div>
+        <div className="container">
+            <img className={"mx-auto ".concat(className)}
+                src={src}
+                alt={alt}>
+            </img>
```
Comment (grandpariley) [inline components/image_centered.tsx:22]: this works for now but I'd like to use https://nextjs.org/docs/basic-features/image-optimization if possible!

File: components/logo.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    ...rest
+  }: ILogo) => {
+    return (
+        <div>
+        <div className="w-screen h-1/6"></div>
+        <div className="container">
+            <img className="w-7/12 mx-auto mb-10"
+                src={src}
+                alt="turnkey_logo">
+            </img>
```
Comment (grandpariley) [inline components/logo.tsx:18]: (same as above)

File: components/pulseLottie.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React, { HTMLAttributes } from "react";
+import LottieView from 'react-lottie';
+import * as animationData from "../public/assets/lotties/pulse-selector.json";
```
Comment (grandpariley) [inline components/pulseLottie.tsx:3]: oooooh fancy

File: pages/login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        const obs = authApi.registerNewToken({ body }).subscribe({
+            next: (tk) => {
+                console.log(tk)
+                router.push({
+                    pathname: "/renter",
+                    query: { userId: tk.id, token: tk.token },
+                })
+            },
+            error: () => console.log(error)
+        });
```
Comment (grandpariley) [inline pages/login/index.tsx:43]: so I'm thinking for this one what we should do is create a nextjs pages endpoint in the pages folder that accesses the secret as an environment variable using https://nextjs.org/docs/basic-features/environment-variables kind of structure

File: components/image_centered.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React, { HTMLAttributes } from "react";
+
+export interface ICenteredImage extends HTMLAttributes<HTMLElement> {
+    src: string;
+    className?: string;
+    alt?: string;
+  }
+  
+  export const CenterdImage = ({
```
Comment (grandpariley) [inline components/image_centered.tsx:9]: spelling mistake :smile:

File: pages/signup/index.tsx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        const obs = authApi.registerNewToken({ body }).subscribe({
+            next: (tk) => {
+                console.log(tk)
+                router.push({
+                    pathname: "/renter",
+                    query: { userId: tk.id, token: tk.token },
+                })
+            },
+            error: () => console.log(error)
+        });
```
Comment (grandpariley) [inline pages/signup/index.tsx:44]: same as the login for this call

**Review by grandpariley — COMMENTED** [review]
The images can stay in here for now but once we get digital ocean spaces up and running they can move there and be referenced by a link!

Overall it looks good - I like the way you're using reusable components and I appreciate how little CSS you've needed to add!

File: components/image_centered.tsx (hunk 5 above)
Comment (matthew-buglass) [reply components/image_centered.tsx:9]: 🙃🙃

**Comment by matthew-buglass** [comment]
Made all the requested changes and removed some print statements to were used for testing

**Pull request closed without merging**